src/border_agent/border_agent.cpp

```diff
@@ -146,6 +147,8 @@ BorderAgent::BorderAgent(otbr::Ncp::ControllerOpenThread &aNcp, Mdns::Publisher
     , mBaseServiceInstanceName(OTBR_MESHCOP_SERVICE_INSTANCE_NAME)
 {
     mNcp.AddThreadStateChangedCallback([this](otChangedFlags aFlags) { HandleThreadStateChanged(aFlags); });
+
+    otBorderAgentSetEphemeralKeyCallback(mNcp.GetInstance(), BorderAgent::HandleEpskcStateChanged, this);
```

Comment:
At this time, mNcp is not initialized yet. It's causing null pointer issue.

Reply:
Move to OTBR start, as mNcp.GetInstance met error when OTBR has not started.